examples/host-function-library/proto/greet_host.pb.go

```diff
@@ -102,6 +104,14 @@ func (p *GreeterPlugin) Load(ctx context.Context, pluginPath string, hostFunctio
 		return nil, err
 	}
 
+	return p.LoadFromBinary(ctx, bytes.NewReader(b), hostFunctions)
+}
+func (p *GreeterPlugin) LoadFromBinary(ctx context.Context, reader io.Reader, hostFunctions HostFunctions) (greeter, error) {
```

Comment:
@brennanjl what do you think about to add a simple test for the ` LoadFromBinary` function?